src/views/Terms/Term.jsx

```diff
@@ -18,14 +19,30 @@ const Term = ({ payload }) => {
 		pronunciation,
 	} = payload;
 
+	const tracking = useTracking();
+
+	const handleTermLinkClick = () => {
+		const idOrName = prettyUrlName ? prettyUrlName : termId;
+
+			tracking.trackEvent({
+					type: 'Other',
+					event: 'GlossaryApp:Other:ResultClick',
+					linkName: 'TermsDictionaryResults',
+					resultIndex: resultIndex + 1,
+					resultIdOrName: idOrName,
+					resultName: termName
+			});
+	};
+
 	return (
 		<div>
 			<dt>
 				<dfn data-cdr-id={termId}>
 					<Link
 						to={DefinitionPath({
 							idOrName: prettyUrlName ? prettyUrlName : termId,
-						})}>
+						})}
+						onClick={() => handleTermLinkClick({}) }>
```

Comment:
you shouldn't need to bind this on click as handleTermLinkClick is a fat arrow.  `onClick={handleTermLinkClick()}` should work